mhlib/mhConfig.go

```diff
@@ -27,6 +27,7 @@ type MHConfig struct {
 	Simulate      bool     `yaml:"simulate"`
 	TargetContext string   `yaml:"targetContext"`
 	Team          string   `yaml:"team"`
+	SETValues     []string `yaml:"setValues"`
```

Comment:
One more thing @prabdeb , please remove the yaml tag here, I don't think it should be possible to set those values in a MH config file.